downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc

```diff
@@ -2,11 +2,12 @@
 
 [id="ref-standlone-platform-inventory_{context}"]
 
-= Example {PlatformName} inventory file with an external managed database
+= Example inventory file for a database on the automation controller node or a non-installer managed database
 
 
 [role="_abstract"]
-This example describes how you can populate the inventory file to install {PlatformName}. This installation inventory file includes both {ControllerName} and {HubName} with a database on the {ControllerName} node or non-installer managed database.
+This example describes how you can populate the inventory file to install {PlatformName}. 
```

Comment:
```suggestion
This example illustrates how to build an inventory file when installing {PlatformName}. 
```

Reply:
I don't understand why this has to be changed. This sentence, in my opinion, is totally correct right now: 
This example describes how you can populate the inventory file to install {PlatformName}. This installation inventory file includes both {ControllerName} and {HubName} with a database on the {ControllerName} node or non-installer managed database.

Reply:
Thanks for keeping me honest @adonisgarciac I'll delete my suggestion if it doesn't fit.